src/includes/set-fingerprint/database-connection/apple.mdx

```diff
@@ -0,0 +1,29 @@
+```swift
+import Sentry
+import CoreData
+
+SentrySDK.start { options in
+    options.beforeSend = { event in
+        if let error = event.error as NSError? {
```

Comment:
l: cast is redundant
```suggestion
        if let error = event.error  {
```

Reply:
The problem is that `Error` doesn't have a domain, so I cant remove it.